DeAceroDLabs/marketplace-frontend
# feat[DMP-479] Validates email is different from another.

## What is done

[[DMP-479] Navigation arrows on categories](https://deacero.atlassian.net/browse/DMP-479)

## How is done

- Added `validateEmailIsDifferent`  function to `EmailField` component so that it will validate if an email is different from another, if not error message appears.

## How to test

1. Run the project via `npm start`
2. /signup
3. Fill first page with the first email.
4. Fill second page.
5. Fill third page with the same first email and see the message error.
6. Fill third page  with different email and see the message error vanish.
